app/src/Controller/ArticlesController.php

```diff
@@ -0,0 +1,117 @@
+<?php
+declare(strict_types=1);
+
+namespace App\Controller;
+
+/**
+ * Articles Controller
+ *
+ * @method \App\Model\Entity\Article[]|\Cake\Datasource\ResultSetInterface paginate($object = null, array $settings = [])
+ */
+class ArticlesController extends AppController
+{
+    /**
+     * Initialize method.
+     *
+     * @return void
+     */
+    public function initialize(): void
+    {
+        parent::initialize();
+
+        $this->loadComponent('Paginator');
+        $this->loadComponent('Flash'); // Include the FlashComponent
+    }
```

Comment:
Please refer https://book.cakephp.org/4/en/controllers.html#request-life-cycle-callbacks to understand about CakePHP's controller request life-cycle callbacks.

Reply:
as on the previous comment I removed the paginator because it was loading twice